context/app/static/js/components/workspaces/WorkspacesAuthenticated.jsx

```diff
@@ -5,30 +5,23 @@ import { AppContext } from 'js/components/Providers';
 import Description from 'js/shared-styles/sections/Description';
 import WorkspacesList from './WorkspacesList';
 
-function AuthenticatedWorkspacesDescription() {
+function WorkspacesAuthenticated() {
   const { workspacesToken } = useContext(AppContext);
 
-  const contact = (
-    <>
-      For any problems, notify Juan Puerto in <code>#workspaces</code>.
-    </>
-  );
-
   if (!workspacesToken) {
-    return (
-      <Description padding="20px">
-        The workspaces token request failed at login. {contact}{' '}
-        <OutboundIconLink href="https://github.com/hubmapconsortium/user_workspaces_server/issues/58">
-          Github issue
-        </OutboundIconLink>
-      </Description>
-    );
+    return <Description padding="20px">The workspaces token request failed at login.</Description>;
   }
 
   return (
     <>
       <Description padding="20px">
```

Comment:
I think it'd be more fitting to use an `Alert` as we have for other warnings, but since this is temporary I don't have a strong preference.

Reply:
As I'm thinking about it now, I think I just want to throw an exception: This is unrecoverable, and it's not a user error.